xml/decoders/Lenz_51.xml

```diff
@@ -113,70 +170,221 @@
       <variable CV="60" item="DC Brake Momentum" mask="XXXXXVXX" default="0" comment="Governs how loco acts in DC track block">
         <enumVal>
           <enumChoice choice="Loco runs under DC analog voltage control">
             <choice>Loco runs under DC analog voltage control</choice>
             <choice xml:lang="it">Loco usa Tensione in Analogico</choice>
-            <choice xml:lang="de">Loco runs under DC analog voltage control</choice>
+            <choice xml:lang="de">Lok läuft unter DC-Analogspannung</choice>
             <choice xml:lang="cs">Lokomotiva používá pro řízení analogové stejnosměrné napětí</choice>
+            <choice xml:lang="nl">Loc rijd onder DC analoge spanning</choice>
           </enumChoice>
           <enumChoice choice="Loco brakes to a stop using momentum set in CV4">
             <choice>Loco brakes to a stop using momentum set in CV4</choice>
             <choice xml:lang="it">Loco frena fino a fermarsi usando Inerzia impostata in CV4</choice>
-            <choice xml:lang="de">Loco brakes to a stop using momentum set in CV4</choice>
+            <choice xml:lang="de">Lok bremst tot stilstand mit den Wert aus CV4</choice>
             <choice xml:lang="cs">Lokomotiva při brždění do zastavení používá zpomalení nastavené v CV4</choice>
+            <choice xml:lang="nl">Loc remt tot stilstand met behulp van de ingestelde waarde uit CV4</choice>
           </enumChoice>
         </enumVal>
         <label>Brake Momentum on DC</label>
         <label xml:lang="it">Inerzia Frenata in Analogico</label>
-        <comment>Governs how loco acts in DC track block</comment>
-        <comment xml:lang="it">Controlla il comportamento della loco in un blocco Analogico</comment>
-        <label xml:lang="de">DC Brake Momentum</label>
-        <comment xml:lang="de">Governs how loco acts in DC track block</comment>
+        <label xml:lang="de">Brems verhalten beim DC-Betrieb.</label>
         <label xml:lang="cs">DC brzdné zpomalení</label>
-        <comment xml:lang="cs">Určuje jak bude lokomotiva reagovat na DC kolejovém obvodu</comment>
+        <label xml:lang="nl">Rem gedrag bij DC-betrieb.</label>
+        
+        <tooltip>&lt;html&gt;Controls how the locomotive reacts in a DC track block.&lt;br&gt;&lt;/html&gt;</tooltip>
+        <tooltip xml:lang="de">&lt;html&gt;Steuert, wie die Lokomotive in einem DC Gleisabschnitt reagiert&lt;br&gt;&lt;/html&gt;</tooltip>
+        <tooltip xml:lang="nl">&lt;html&gt;Regelt hoe de loc reageert in een DC spoor sectie.&lt;br&gt;&lt;/html&gt;</tooltip>
+      </variable>
+      <!-- No compliant Speed Table -->
+      <variable CV="67" item="Speed Table" comment="Range 1-63">
+        <speedTableVal min="1" max="63" entries="15"/>
+        <label>                                                     Speed Table (CV67-81) (1-63)</label>
+        <label xml:lang="cs">                                                             Rychlostní tabulka (CV67-81) (1-63)</label>
```

Comment:
No need to have translated empty labels, the default will be used for other languages

Reply:
Sorry but I don't see any empty labels